src/random.cpp

```diff
@@ -297,7 +315,12 @@ void SeedHardwareSlow(CSHA512& hasher) noexcept {
 #elif defined(__aarch64__) && defined(HWCAP2_RNG)
     if (g_rndr_supported) {
         for (int i = 0; i < 4; ++i) {
-            uint64_t out = GetRNDRRS();
+            uint64_t out;
+            if (g_rndrrs_supported) {
+                out = GetRNDRRS();
+            } else {
+                out = GetRNDR();
+            }
```

Comment:
As we already use `GetRNDR` in `SeedHardwareFast` which is called way more often than `SeedHardwareSlow` (including *in* `SeedSlow`) i'm not sure if calling it here has any additional value. Let's just skip this loop if rndrrs isn't supported.

Reply:
Thank you, updated.